pkg/package-server/server/server.go

```diff
@@ -52,7 +52,7 @@ func NewCommandStartPackageServer(ctx context.Context, defaults *PackageServerOp
 	}
 
 	flags := cmd.Flags()
-	flags.DurationVar(&defaults.SyncInterval, "interval", defaults.SyncInterval, "interval at which to re-sync CatalogSources")
+	flags.DurationVar(&defaults.DefaultSyncInterval, "interval", defaults.DefaultSyncInterval, "default interval at which to re-sync CatalogSources")
```

Comment:
Argh, defaults.DefaultSyncInterval :P

Reply:
It makes more sense when the structure is not referenced as `defaults`.